### PR TITLE
Sammantha

### PR DESCRIPTION
**Pull request by sammiefrog**

### CODE REVIEW

File: client/sammantha.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            <div class="image">
+                <img src="/images/sammantha.jpg">
+            </div>
+            <div class="content">
+                <div class="header">Sammantha</div>
+                <div class="meta">
+                    <a>Coworker</a>
+                </div>
+                <div class="description">
+                    I recently graduated the Coding bootcamp at Vanderbilt University. I have a background in Hospitality, and a Bachelors of Science in Psychology & Neuroscience. In my free time I enjoy cooking, hiking, practicing yoga, and watching netflix with my cat Bobbie.
```
Comment (ravens-eye) [inline client/sammantha.html]: This should really be in a `<p>` to keep things semantic :grinning:

File: client/sammantha.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sammantha Sasenick</title>
+    <!-- Semantic UI -->
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js" crossorigin="anonymous"></script>
```
Comment (ravens-eye) [inline client/sammantha.html]: No need to load in js you're not using! Same with the semantic.min.js below, and jQuery at the bottom of the file.

**Review by ravens-eye — CHANGES_REQUESTED** [review]

File: client/sammantha.html (hunk 1 above)
Comment (sammiefrog) [reply client/sammantha.html]: Got it, sorry! I just pushed the changes.

File: client/sammantha.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sammantha Sasenick</title>
+    <!-- Semantic UI -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css">
+    <script src="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.js"></script>
```
Comment (ravens-eye) [inline client/sammantha.html]: This one can be removed too, I don't see any JavaScript necessities

**Review by ravens-eye — CHANGES_REQUESTED** [review]

File: client/sammantha.html (hunk 1 above)
Comment (ravens-eye) [reply client/sammantha.html]: No need to apologize, I'm a very harsh PR reviewer. :)

File: client/sammantha.html (hunk 3 above)
Comment (sammiefrog) [reply client/sammantha.html]: done

**Review by ravens-eye — APPROVED** [review]

**Pull request merged**